azure-pipelines.yml

```diff
@@ -92,6 +85,41 @@ steps:
       command: push
       tags: latest
 
+  - task: Bash@3
+    name: mtrx
+    displayName: 'set up functional test matrix variable'
+    inputs:
+      targetType: filePath
+      filePath: ./script/generate_matrix.sh
+
+- job: FunctionalTesting
+  pool:
+    vmImage: 'ubuntu-16.04'
+  dependsOn: BuildAndNonBrowserTesting
+  strategy:
+    maxParallel: 5
+    matrix: $[ dependencies.BuildAndNonBrowserTesting.outputs['mtrx.legs'] ]
```

Comment:
Current policy is `master` runs both FF and Chrome cucumber tests, branches run only Chrome

My reading of this is that the functional tests get run on both Chrome and FF on the branches? Can we make this run only Chrome tests on a branch?

Reply:
A parallel job is initiated but the task within it has a condition that is only run on Chrome for a branch and Chrome and Firefox for master,